pkg/sync/v13/data/DaemonSet.apps/openshift-azure-logging/log-analytics-node-agent.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     metadata:
       annotations:
         agentVersion: 1.10.0.1
-        dockerProviderVersion: 7.0.0-6
+        dockerProviderVersion: 8.0.0-1
```

Comment:
Why do we change old release this? This will invalidate old hash and trigger rotation?

Reply:
I think this is causing you failing tests as hash changed and our mocked cluster upgrade test fails. They are not very verbose in the test output

Reply:
thank you for catching this. v13 was indeed mistake. I will fix it